<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/newPrefix/Images/StartSession.png);</string>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="RecordPart1_Btn">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>374</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 67, 101);
background-image: url();
border-radius: 15px;
font-size: 12px;
color: rgb(255, 252, 255)</string>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="RecordPart1_Txt">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>328</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color:rgb(255, 252, 255);
font-size: 10px;</string>
    </property>
    <property name="text">
     <string>0:00/0:30</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{background-image: url();}
QSlider::groove:horizontal { 
	background-color: rgb(255, 255, 0);
	border: 0px solid #424242; 
	height: 4px; 
	border-radius: 6px;
}
QSlider::handle:horizontal { 
	background-color: rgb(0, 0, 255); 
	border: 2px solid rgb(0, 0, 255); 
	width: 8px; 
	height: 10px; 
	line-height: 20px; 
	margin-top: -5px; 
	margin-bottom: -5px; 
	border-radius: 6px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 6px; 
}
QSlider::add-page:horizontal {
    background: white;
}

QSlider::sub-page:horizontal {
    background: pink;
}</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
